core/src/main/java/io/grpc/MethodDescriptor.java

```diff
@@ -252,7 +252,8 @@ public boolean isSafe() {
 
   /**
    * Setting safe on the method. Works only with unary rpcs.
-   * A safe request does nothing expect retrieval so it has no side effects on the server side.
+   *
+   * <p>A safe request does nothing except retrieval so it has no side effects on the server side.
```

Comment:
I had meant on the `isSafe`  That way users don't have to read the comment for `withSafe` to know what `isSafe` means.